src/platform/silabs/DiagnosticDataProviderImpl.cpp

```diff
@@ -30,6 +30,7 @@
 #endif
 #include "FreeRTOS.h"
 #include "sl_memory_manager.h"
+#include "task.h"
```

Comment:
same comment

Reply:
Yes it is needed this file still uses FreeRTOS api and we are currently still using it. Previously it had access to task.h through genericPlatformImpl_FreeRTOS.h but it isn't he case anymore for our platform. Else there is a failure.